### PR TITLE
Make the error handle deal with remap redirects

### PR DESCRIPTION
**Pull request by zwoop**

### CODE REVIEW

**Comment by zwoop** [comment]
[Approve CI]

**Comment by zwoop** [comment]
[approve ci].

**Comment by bryancall** [comment]
[approve ci rocky]

**Comment by bryancall** [comment]
[approve ci autest]

File: include/cripts/Error.hpp [hunk 1]
```diff
@@ -111,9 +111,22 @@ class Error
   }
 
   void
-  Fail()
+  Fail(bool redirect = false)
```
Comment (bneradt) [inline include/cripts/Error.hpp:114]: Is this parameter ever used?

**Review by bneradt — CHANGES_REQUESTED** [review]

File: include/cripts/Error.hpp (hunk 1 above)
Comment (zwoop) [reply include/cripts/Error.hpp:114]: It could be used in a Cript.

Comment (bneradt) [reply include/cripts/Error.hpp:114]: Ah, OK. Makes sense.

**Review by cmcfarlen — APPROVED** [review]

**Review by cmcfarlen — APPROVED** [review]

**Comment by cmcfarlen** [comment]
Cherry-picked to v10.0.x

**Pull request merged**